src/reducers/project-text-search.js

```diff
@@ -90,7 +91,11 @@ function update(
   return state;
 }
 
-type OuterState = { projectTextSearch: ProjectTextSearchState };
+type OuterState = { projectTextSearch: Record<ProjectTextSearchState> };
```

Comment:
I think it's not a record anymore.

Reply:
I wasn't able to use `getState()` as an argument to the selectors without changing this.

```javascript
// in src/reducers/types.js
export type State = {
  ...
  projectTextSearch: Record<ProjectTextSearchState>,
  ...
};
```

Reply:
That's strange

Reply:
I think we need to remove `Record` from here https://github.com/devtools-html/debugger.html/blob/master/src/reducers/types.js#L31